docs/octopus_deploy_worker_pool.md

```diff
@@ -0,0 +1,24 @@
+# octopus_deploy_worker_pool
+
+Describes an Octopus Deploy [Worker Pool](https://octopus.com/docs/infrastructure/workers/worker-pools) resource.
+
+## Example
+
+```ruby
+describe octopus_deploy_worker_pool('My Worker Pool').in_space("My Space") do
+  it { should exist } 
+end
+```
+
+#### Type
+
+```ruby
```

Comment:
should this have the link to the auth doc?

Reply:
Added